pennylane/templates/subroutines/select.py

```diff
@@ -25,26 +25,34 @@
 
 
 class Select(Operation):
-    r"""Applies specific input operations depending on the state of
-    the designated control qubits.
+    r"""Applies individual operations depending on the state of
```

Comment:
Is this easier/plainer language? (not a required change)
```suggestion
    r"""Applies different operations depending on the state of
```